lib/jsrt/Jsrt.cpp

```diff
@@ -2552,8 +2552,11 @@ STDAPI_(JsErrorCode) JsRunWasmScript(_In_z_ const wchar_t * script, _In_ JsSourc
         *result = scriptFunction;
         return JsNoError;
     });
-}
+#else
+    result = nullptr; // Satisfy code analysis
```

Comment:
I think this might fail on some code analysis since the _Out_ annotation means data will be passed out in that parameter, and the change of the pointer itself will not propagate to the caller. Perhaps assign this a real `JsValueRef` which corresponds to a null return? @MikeHolman is there a singleton global or something like that for this purpose to avoid an unnecessary allocation?